src/main/MainScreenContainer.js

```diff
@@ -31,12 +51,40 @@ class MainScreenContainer extends React.Component {
 
   render() {
     return (
-      <MainScreen
-        fetchOlder={this.fetchOlder}
-        fetchNewer={this.fetchNewer}
-        doNarrow={this.doNarrow}
-        {...this.props}
-      />
+      <View style={styles.screen}>
+        <MainScreen
+          fetchOlder={this.fetchOlder}
+          fetchNewer={this.fetchNewer}
+          doNarrow={this.doNarrow}
+          openPopUp={this.openPopUp}
+          {...this.props}
+        />
+        <Modal
```

Comment:
This shouldn't be in MainScreenContainer because it only concerns a specific screen. If we push a new screen onto the Navigator to display the names, we won't need to add this